app/services/location_lookup_service.rb

```diff
@@ -49,7 +49,9 @@ def areas
 
   def response
     @response ||= begin
-                    JSON.parse(mapit.location_for_postcode(postcode).to_json)
+                    Rails.cache.fetch(postcode, expires_in: 3.hours) do
```

Comment:
I'd give this a slightly more distinct key, perhaps "mapit-location-#{postcode}".

Reply:
makes sense, I that just to prevent potential clashes, or is there somewhere that we regularly read out of the cache and so would want to know what this "postcode" thing actually is?

Reply:
Yeah just to prevent any potential clashes. It doesn't seem like the only thing we could cache with a postcode.